google/cloud/firestore_v1/async_document.py

```diff
@@ -353,31 +357,34 @@ async def get(
                 :attr:`create_time` attributes will all be ``None`` and
                 its :attr:`exists` attribute will be ``False``.
         """
-        request, kwargs = self._prep_get(field_paths, transaction, retry, timeout)
+        from google.cloud.firestore_v1.base_client import _parse_batch_get
```

Comment:
is this different than prep_batch_get?

Reply:
Yes, `prep_` is what sets up the request. `_parse` is what turns the response into a plain `DocumentSnapshot`.